include/dynd/types/tuple_type.hpp

```diff
@@ -145,13 +147,13 @@ namespace ndt {
     }
 
     /** Makes a tuple type with the specified types */
-    static type make(const nd::array &field_types, bool variadic = false)
+    static type make(std::vector<type> field_types, bool variadic = false)
     {
       return type(new tuple_type(field_types, variadic), false);
```

Comment:
Ah yes, for returning vectors from e.g. get_field_types() I have const std::vector & everywhere now, for constructing tuples I have const std::vector.